infra/terraform/templates/launch.tf

```diff
@@ -59,3 +59,8 @@ resource "aws_autoscaling_group" "wellcomecollection_ecs_asg" {
   vpc_zone_identifier       = ["${aws_subnet.public_a.id}", "${aws_subnet.public_b.id}"]
   target_group_arns         = ["${aws_alb_target_group.wellcomecollection.id}"]
 }
+
+output "ami_id" {
+  value = "${data.aws_ami.ecs_optimized.id}"
```

Comment:
It basically comes with the ECS service installed. 
[You then just need to set which cluster it should join](https://github.com/wellcometrust/wellcomecollection.org/blob/master/infra/terraform/templates/launch.tf#L46) to get it up and running.

As ECS is running docker - it's the docker container you need to keep up to date.